bika/lims/jsonapi/__init__.py

```diff
@@ -129,10 +130,15 @@ def load_field_values(instance, include_fields):
 def get_include_methods(request):
     """Retrieve include_methods values from the request
     """
-    methods = request.get("include_methods", "")
-    include_methods = [
-        x.strip() for x in methods.split(",") if x.strip()]
-    return include_methods
+    include_methods = request.get("include_methods[]")
+    if not include_methods:
+        include_methods = request.get("include_methods", [])
```

Comment:
When is the variable named `include_methods[]` in the request?

Reply:
First of all, `include_methods` is not used as a parameter anywhere in the code, but I was playing around and found this issue. Nevertheless, `include_fields` from here:

https://github.com/senaite/senaite.core/blob/699f02f950c86ac650ddcf86da0f1085d7a2e7de/bika/lims/browser/js/coffee/bika_widgets/remarkswidget.coffee#L121-L129

is received as `include_fields[]`. Therefore, this trick here:

https://github.com/senaite/senaite.core/blob/699f02f950c86ac650ddcf86da0f1085d7a2e7de/bika/lims/jsonapi/__init__.py#L56-L58

Same happens if you use `include_methods` instead. Thus, I just followed same criteria and did not go any further. As a side note, the issue this PR addresses happens with `include_fields` to, but is a hidden "bug" because of:

https://github.com/senaite/senaite.core/blob/699f02f950c86ac650ddcf86da0f1085d7a2e7de/bika/lims/jsonapi/__init__.py#L88-L89

and this code will work even if `include_fields` is a string instead of a list.

Reply:
Ah, I think I got it. This means that either JQuery or the Zope Publisher is converting `include_methods` -> `include_methods[]`?

Anyhow, going to merge then.